templates/rapidsai/partials/clone_and_build_blazing.j2

```diff
@@ -0,0 +1,39 @@
+{# This partial clones the BlazingSQL repo and build and installs it into the rapids conda env. #}
+
+{# Install build prerequisites #}
+RUN gpuci_conda_retry install -y -n rapids \
+    blazingsql-build-env=${RAPIDS_VER} \
+    blazingsql-notebook-env=${RAPIDS_VER}
+
+ENV CUDF_HOME=/rapids/cudf
+
+# Clone, build, install
+RUN mkdir -p ${BLAZING_DIR} \
+    && cd ${BLAZING_DIR} \
+    && git clone -b branch-${RAPIDS_VER} https://github.com/BlazingDB/blazingsql.git
```

Comment:
```suggestion
    && git clone -b ${BUILD_BRANCH} https://github.com/BlazingDB/blazingsql.git
```

We'll need to use `${BUILD_BRANCH}` here instead of `branch-*` so that we can build off the `main` branch when necessary. This is important since our hotfixes target the `main` branch. So if there are any post-release issues that require a hotfix, we can rebuild from `main` and our images will include the hotfix. Using `${BUILD_BRANCH}` will require the lines below to be added to `templates/rapidsai/Devel.dockerfile.j2`. See #207 for more details.

```dockerfile
ARG RAPIDS_VER
ARG BUILD_BRANCH="branch-${RAPIDS_VER}"
```